# This message is a submessage of PhantomIntelligenceFrame and
# is not intended to be used separately.

uint16 id
PhantomIntelligenceTrack[] tracks
